opm/parser/eclipse/Units/UnitSystem.cpp

```diff
@@ -129,6 +138,9 @@ namespace {
         1, /* gas-oil ratio */
```

Comment:
Fair enough - they might be buggy for field (but should be fine for lab and metric).